CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Main (unreleased)
 
+## v235 (2022/01/03)
```

Comment:
Even more reasonable 

```suggestion
## v235 (2022-01-03)
```

Reply:
We have informally settled on slashes https://github.com/heroku/buildpacks-jvm/blob/main/buildpacks/jvm/CHANGELOG.md